article/_posts/2019-10-15-multi-view-shape-priors.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: review
+title: "Learning shape priors for robust cardiac MR segmentation from multi-view images"
+tags: deep-learning segmentation autoencoder
+author: "Nathan Painchaud"
+cite:
+    authors: "Chen Chen, Carlo Biffi, Giacomo Tarroni, Steffen Petersen, Wenjia Bai, Daniel Rueckert"
+    title:   "Learning shape priors for robust cardiac MR segmentation from multi-view images"
+    venue:   "MICCAI 2019"
+pdf: "https://arxiv.org/pdf/1907.09983.pdf"
+---
+
+
+# Highlights
+- Novel autoencoder architecture (Shape MAE) which learns latent representation of cardiac shapes from multiple
+  standard views
+- Multi-view UNet that can incorporate anatomical shape priors to guide the segmentation of short-axis images
+- More robust than 2D or 3D segmentation baselines when using less training data (10%)
+
+
+# Introduction
+A number of methods already improve the robustness of cardiac segmentations, but they have some drawbacks:
+- Heavy computational cost (ensemble model)
+  or
+- Requirement of fully annotated 3D high-resolution images, without inter-slice motion artifacts (often not feasible
+  for patients with cardiovascular diseases)
+
+
+# Methods
+Given a source view $$X_i$$, the network learns the low-dimensional representation $$z_i$$ of $$X_i$$ that best
+reconstructs all the $$j$$ target views segmentations $$Y_j$$.
+
+The authors use four source views $$X_i$$ ($$i = 1, ..., 4$$) which are three LA images - the two-chamber view
+(LA1), three-chamber view (LA2), the four-chamber view (LA3) - and one mid-ventricular slice (Mid-V) from the SA view.
+
+The target segmentations views $$Y_j$$ ($$j = 1, ..., 6$$) correspond to the four previous views plus two SA slices:
+the apical one and the basal one.
+
+The Shape MAE architecture is detailed in Figure 1.
+
+![](/article/images/MultiViewShapePriors/figure1.png)
+
+The global loss for training the Shape MAE is shown in equation 1.
+
+![](/article/images/MultiViewShapePriors/equation1.png)
+
+where:
+- $$L_intra$$ denotes the segmentation loss when the source view $$X_i$$ and the target view $$Y_j$$
+- $$L_inter$$ denotes the loss when two views are differentthe number of filters at each level by four times to account
```

Comment:
(...) denotes the loss when two views are differentthe number of filters at each level (...) :
(...) denotes the loss when two views differ in the number of filters at each level (...) ?

Reply:
Well, that's a bit of an embarrassing typo... Thanks for the catch! I applied your proposed formulation.

Reply:
Done in e59ffce.